handbook/engineering/search/index.md

```diff
@@ -1,3 +1,29 @@
-# Search
+# Search team
 
-TODO: identify an owner to fill out this page and update CODEOWNERS
\ No newline at end of file
+## Vision
+
+Build and deliver Universal Code Search to all developers.
+
+## Contact
+
+- #search channel or @searchers on Slack.
+- [@sourcegraph/search](https://github.com/orgs/sourcegraph/teams/search) team or [search label](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+is%3Aopen+label%3Asearch) on GitHub.
+
+## Project Leads
+
+#### Technical lead: [Keegan](https://github.com/keegancsmith)
```

Comment:
I would like to avoid giving these explicit names to avoid confusion. My suggestion is to remove the heading and be more descriptive, like this:

```
[Keegan](https://about.sourcegraph.com/company/team#keegan-carruthers-smith) owns technical responsibilities:

- the list

[Farhan](https://about.sourcegraph.com/company/team#) owns organizational responsibilities:

- the list